### DISCUSSION TITLE
chromedriver service

### DISCUSSION THREAD

**Discussion by cg-tester** [webdriverio/webdriverio#11051] [Q&A]
Hi everyone,

I want to use chromedriver service so I add ' chromedriver ' in services array in config file.
then below error log !  of course I npm installed wdio-chromedriver-service.  Could you help me to fix it ?

A service failed in the 'onPrepare' hook
SevereServiceError: Cannot find package 'chromedriver'

**Comment by christian-bromann** [comment]
> Could you help me to fix it ?

With the latest version of WebdriverIO there is no need to use the Chromedriver service anymore. WebdriverIO manages driver for you.
